gammapy/data/observation_summary.py

```diff
@@ -0,0 +1,107 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import SkyCoord
+
+from .observation import ObservationTable
+
+__all__ = [
+    'ObservationTableSummary',
+]
+
+class ObservationTableSummary(object):
+    """Observation table summary.
+
+    Class allowing to summarize informations conatained in 
+    Observation index table (`~gammapy.data.ObservationTable`)
+
+    Parameters:
+    -----------
+    obs_table:  `~gammapy.data.ObservationTable`
+        Observation index table
+    target_pos: `~astropy.coordinates.SkyCoord`
+        Target position
+    """
+
+    def __init__(self, obs_table, target_pos=None):
+        self.obs_table = obs_table
+        self.target_pos = target_pos
+
+    def get_offset(self):
```

Comment:
Maybe make this private (by adding an underscore), i.e. not part of the public API?
And this is nicer as a property:
```python
@property
def _offset(self):
    pnt_pos = SkyCoord(self.obs_table['RA_PNT'], self.obs_table['DEC_PNT'], unit='deg')
    return pnt_pos.separation(self.target_pos)
```